ksql-functional-tests/src/main/java/io/confluent/ksql/test/tools/Record.java

```diff
@@ -37,7 +37,7 @@
   private final Object value;
   private final Optional<Long> timestamp;
   private final WindowData window;
-  private JsonNode jsonValue;
+  private Optional<JsonNode> jsonValue;
```

Comment:
I take it were making this change as its valid to have a null value in a stream?

Reply:
no - there are parts of the code that create a `Record` from the actual value without a corresponding spec, those cases will not have the original jsonValue